test/unit/specs/field-types/PseudonymRegistrationComponent.spec.js

```diff
@@ -0,0 +1,243 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-unused-vars */
+import PseudonymRegistrationComponent from '@/components/field-types/PseudonymRegistrationComponent'
+import { mount } from 'vue-test-utils'
+import td from 'testdouble'
+import pseudonymRegistration from '@/util/helpers/pseudonymRegistration'
+import Vue from 'vue'
+
+const config = {
+  'ID': 'PseudonymRegistration',
+  'GeneratedTokenDescription': 'MyPseudonymID is cool for lost of reasons',
+  'GeneratedTokenName': 'MyPseudonymID',
+  'LinkEntityName': 'PseudonymConnector',
+  'FieldName': 'OriginalID'
+}
+
+const PseudonymRegistrationResponse = {
+  'items': [
+    config
+  ]
+}
+const pseudonymID = 'PseudonymID'
+const requestConfiguration = td.function('pseudonymRegistration.requestConfiguration')
+const submitPseudonymRegistration = td.function('pseudonymRegistration.submitPseudonymRegistration')
+
+describe('PseudonymRegistrationComponent unit tests', () => {
+  const mockParentFunction = () => {
+    return null
+  }
+
+  const field = {
+    id: 'PseudonymRegistration',
+    label: 'Pseudonym Registration',
+    description: 'Pseudonym Registration description',
+    type: 'ExistingKey',
+    disabled: false
+  }
+
+  const fieldState = {
+    showOptionalFields: false,
+    $touched: false,
+    $submitted: false,
+    $invalid: false,
+    _addControl: mockParentFunction
+  }
+
+  describe('PseudonymRegistrationComponent with existing record', () => {
+    const propsData = {
+      value: 'hallo',
+      field: field,
+      fieldState: fieldState,
+      isRequired: true,
+      isValid: false,
+      inputDebounceTime: 0
+    }
+
+    let wrapper
+    beforeEach(() => {
+      td.reset()
+      const requestConfiguration = td.function('pseudonymRegistration.requestConfiguration')
+      td.when(requestConfiguration('PseudonymRegistration')).thenResolve(PseudonymRegistrationResponse)
+      td.replace(pseudonymRegistration, 'requestConfiguration', requestConfiguration)
+
+      wrapper = mount(PseudonymRegistrationComponent,
+        {
+          propsData: propsData,
+          stubs: { 'formFieldMessages': '<div class="form-control-feedback"><div class="invalid-feedback">This field is required</div></div>' }
+        }
+      )
+    })
+
+    it('should load the component with "PseudonymRegistrationComponent" as a name', () => {
+      expect(PseudonymRegistrationComponent.name).to.equal('PseudonymRegistrationComponent')
+    })
+    it('renders correctly with minimal props', () => {
+      expect(wrapper.contains('input')).to.equal(true)
+      expect(wrapper.contains('label')).to.equal(true)
+      expect(wrapper.contains('small')).to.equal(true)
+    })
+    it('should load default data with the help of props', () => {
+      expect(wrapper.vm.localValue).to.equal('hallo')
+    })
+
+    it('should render the label correctly', () => {
```

Comment:
We're not really consistent at placing enters between its (I prefer having an enter between them)